### DISCUSSION TITLE
New tenant name could be "default"

### DISCUSSION THREAD

**Discussion by LiuBodong** [apache/dolphinscheduler#6259] [General]
this discussion is from issue #6220

**Comment by lenboo** [comment]
- I think the 'default'(value=-1) cannot be removed. 

You should must select one tenant when saving DAG if 'default' is removed.
I don't think this operation is as simple as 'default '.
using 'default', the same DAG can be run different tenants when multiple users run one DAG.

- Maybe we can change the name 'default' to another one.
